index.js

```diff
@@ -354,6 +357,8 @@ class Plugin extends MiniAccountsPlugin {
         const writer = new Writer()
         const response = this._prefix + account.getAccount()
         writer.writeVarOctetString(Buffer.from(response))
+        writer.writeUInt8(ASSET_SCALE)
+        writer.writeVarOctetString(Buffer.from(ASSET_CODE, 'utf8'))
```

Comment:
This could call `serializeIldcpResponse` from `ilp-protocol-ildcp` so we don't duplicate the code.